### PR TITLE
[JB][OPS-8697] increase test coverage

### PR DESCRIPTION
**Pull request by barnesjake**

### CODE REVIEW

File: app/model/HeadOfDutyIndicators.scala [hunk 1]
```diff
@@ -16,33 +16,20 @@
...
 import play.api.libs.json.Format
-import play.api.mvc.{PathBindable, QueryStringBindable}
 
 import scala.collection.immutable
 
+sealed abstract class HeadOfDutyIndicator extends EnumEntry
+
 object HeadOfDutyIndicator {
   implicit val format: Format[HeadOfDutyIndicator] = EnumFormat(HeadOfDutyIndicators)
-  implicit val pathBinder: QueryStringBindable[HeadOfDutyIndicator] = bindableA(_.toString)
```
Comment (barnesjake) [inline app/model/HeadOfDutyIndicators.scala:29]: deleted as it isn't used

File: app/model/HeadOfDutyIndicators.scala [hunk 2]
```diff
@@ -16,33 +16,20 @@
...
-  implicit val headOfDutyBinder: PathBindable[HeadOfDutyIndicator] = valueClassBinder(_.toString)
-
-}
-
-sealed abstract class HeadOfDutyIndicator extends EnumEntry {
 }
 
 object HeadOfDutyIndicators extends Enum[HeadOfDutyIndicator] {
 
-  def forCode(code: String): HeadOfDutyIndicator = values.find(_.toString == code) match {
```
Comment (barnesjake) [inline app/model/HeadOfDutyIndicators.scala:39]: deleted as it isn't used

File: app/model/StatusTypes.scala [hunk 3]
```diff
@@ -16,29 +16,22 @@
...
 import play.api.libs.json.Format
-import play.api.mvc.{PathBindable, QueryStringBindable}
 
 import scala.collection.immutable
 
+sealed abstract class StatusType extends EnumEntry
+
 object StatusType {
   implicit val format: Format[StatusType] = EnumFormat(StatusTypes)
-  implicit val pathBinder: QueryStringBindable[StatusType] = bindableA(_.toString)
```
Comment (barnesjake) [inline app/model/StatusTypes.scala:29]: deleted as it isn't used

File: app/model/StatusTypes.scala [hunk 4]
```diff
@@ -16,29 +16,22 @@
...
-
 }
 
-sealed abstract class StatusType extends EnumEntry
-
 object StatusTypes extends Enum[StatusType] {
 
   val values: immutable.IndexedSeq[StatusType] = findValues
 
-  def forCode(code: String): Option[StatusType] = values.find(_.toString == code)
```
Comment (barnesjake) [inline app/model/StatusTypes.scala:40]: deleted as it isn't used

File: app/model/TaxTypes.scala [hunk 5]
```diff
@@ -17,21 +17,15 @@
...
 
 import scala.collection.immutable
 
-sealed abstract class TaxType extends EnumEntry {
-}
+sealed abstract class TaxType extends EnumEntry
 
 object TaxType {
   implicit val format: Format[TaxType] = EnumFormat(TaxTypes)
-  implicit val pathBinder: QueryStringBindable[TaxType] = bindableA(_.toString)
```
Comment (barnesjake) [inline app/model/TaxTypes.scala:32]: deleted as it isn't used

**Review by pawelpanasewicz — APPROVED** [review]

**Review by pawelpanasewicz — APPROVED** [review]

**Pull request merged**